src/software/thunderscope/proto_unix_io.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
         self.unix_senders = {}
         self.unix_listeners = {}
         self.send_proto_to_observer_threads = {}
+        self.stop_running = False
```

Comment:
nit: its slightly more intuitive imo if we flip this variable to `self.running = True`

Reply:
fixed